draft/index.html

```diff
@@ -697,6 +697,66 @@
 
 </section>
 
+<section data-dfn-for="teaches">
+
+### <dfn>teaches</dfn>
+
+Verweis auf Kompetenz(en), die mit Hilfe der beschreibenen Ressource erreicht werden können:
+
+- `id` zur Angabe der ID der Kompetenz (in Form einer URI)
+- `prefLabel`: Zur Angabe einer Bezeichnung für die Kompetenz
```

Comment:
```suggestion
- Über `id` MUSS die ID zur Kompetenz in Form einer IRI angegeben werden.
- Über `prefLabel` soll die Bezeichnung der Kompetenz angegeben werden (Objekt mit Sprachprefix als Schlüssel).
```

Reply:
Wir schreiben bisher überall "URI". Wenn wir das austauschen wollen, sollten wir das leiber in einem gesonderten Ticket machen.

Was die Formulierungen angeht, sollte das im Profil möglichst einheitlich sein. @sroertgen hat sich da m.E. gut an [`mainEntityOfPage`](https://dini-ag-kim.github.io/amb/draft/#mainentityofpage) orientiert. (Was nicht heißen soll, dass wir das – einheitlich – nochmal anpassen könnten.)

Reply:
> Wir schreiben bisher überall "URI". Wenn wir das austauschen wollen, sollten wir das leiber in einem gesonderten Ticket machen.

Gerne, wenn ich es richtig verstehe, muss eine URI nicht immer gleich sein, solange sie auf dieselbe Ressource auflöst.